libs/design/src/atoms/form/select/select.md

```diff
@@ -0,0 +1,10 @@
+# Select Component
+
+`daff-select`  is a form control element that can be used in forms.
+
+
+## Usage
+
+```
+<select daff-select />
```

Comment:
This should have a little more information about its inputs and such. Like
```
<select daff-select [formControl]="myFormControl" [formSubmitted]="myFormSubmitted" />
```
and then maybe describe the inputs

Reply:
Shouldn't this only be covered in design-land?

Reply:
Eventually, when we write more documentation, we'll want to say what the inputs and outputs are. Right now, if someone outside of us three wanted to use @daffodil/design in their own app, they'd need to look into the source code to figure out what the inputs of daff-select are. Given, I think a lot of our stuff is under-documented

Reply:
Updated 31f3871